backend/app/tests/crud/test_token_blacklist_crud.py

```
@@ -0,0 +1,180 @@
+
+@patch("app.crud.select") 
+@patch("app.crud.TokenBlacklist") # is_token_blacklisted uses TokenBlacklist from crud's global scope (if imported there)
+                                 # Re-checking crud.py: is_token_blacklisted does NOT dynamically import.
+                                 # It relies on a module-level TokenBlacklist (which isn't there).
+                                 # This means is_token_blacklisted will likely fail unless TokenBlacklist is imported at module level in crud.py
+                                 # For now, assume it meant to dynamically import, or patch app.models.TokenBlacklist
+                                 # and hope select() gets that one.
+                                 # The current crud.py's is_token_blacklisted will fail with NameError for TokenBlacklist.
+                                 # Assuming it should be `from app.models import TokenBlacklist` implicitly.
+                                 # So, patching app.models.TokenBlacklist is the consistent way.
+@patch("app.models.TokenBlacklist")                                 
```

Ruff (0.11.9)

Trailing whitespace

Remove trailing whitespace

(W291)